src/CommunityVoices/Public/js/error-log-collection.js

```diff
@@ -0,0 +1,97 @@
+const allErrorsApiUrl = '/community-voices/api/errors-log',
+    optionsUrl = window.location.pathname.split('log')[1], // the number of lines from end to include .e.g. /500
+    dataTypeToReceive = "JSON",
+    outerArrayKey = 'errorsLog',
+    errorsDisplayElm = $('#errors'),
+    startDate = "1/1/2010",
+    endDate = moment();
+var data = {},
+    numberOfLines = 0, // tracks how far from end of file to optimize adding lines
+    clkBtn = null,
+    listOfErrors = [];
+
+$(document).ready(function() {
+    data['firstTime'] = true;
```

Comment:
Yeah I still feel like using this variable is kind of weird.

Reply:
See new commit. Took care of this and checked for empty query string instead